File: truss-compiler-maven-plugin/src/main/java/truss/compiler/generator/bound/xml/BoundSuperArgument.java
package truss.compiler.generator.bound.xml;

import javax.xml.bind.annotation.*;

@XmlRootElement(name = "superArgument")
@XmlAccessorType(XmlAccessType.FIELD)
public class BoundSuperArgument {
    @XmlAttribute
    private String name;
    @XmlValue
    private String value;

    public String getName() {
        return name;
    }

    public String getValue() {
        return value;
    }
}

File: Truss.Compiler.Generate/Bound/Xml/BoundSuperArgument.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace Truss.Compiler.Generate.Bound.Xml {
    [XmlRoot("superArgument")]
    public class BoundSuperArgument {
        [XmlAttribute("name")]
        public string Name { get; private set; }

        [XmlAttribute("value")]
        public string Value { get; private set; }
    }
}
